sysl2/sysl/seqs/visitor.go

```diff
@@ -207,17 +232,28 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 			log.Errorln(err)
 		}
 	}()
-
+	var err error
 	switch t := e.(type) {
 	case *EndpointCollectionElement:
-		return v.visitEndpointCollection(t)
+		err = v.visitEndpointCollection(t)
+		for bbKey, bbVal := range t.blackboxes {
+			if bbVal.valueType >= BbApplication && bbVal.visitCount == 0 {
+				var scope string
+				switch bbVal.valueType {
+				case BbApplication:
```

Comment:
Endpoints are yet to be drawn. some of the application level endpoints might not have been hit yet. warning about these at this level is incorrect.

Reply:
Add test case where you are warning about not hitting App level blackbox.